sqld/src/lib.rs

```diff
@@ -145,6 +149,38 @@ async fn run_service(
         });
     }
 
+    match (&config.heartbeat_url, &config.heartbeat_period) {
+        (Some(heartbeat_url), Some(heartbeat_period)) => {
+            tracing::info!(
+                "Server sending heartbeat to URL {} every {}",
+                heartbeat_url,
+                humantime::format_duration(*heartbeat_period).to_string()
```

Comment:
you can use the `Debug` implementation of Duration to get nice formatting: https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=8eca3df4fb1a508658d91db44eb35174

no need for humantime